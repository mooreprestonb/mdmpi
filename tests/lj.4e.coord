<coords>
<natoms> 4 </natoms> <step> 0 </step>
<xyz> 0.999009 0     0           </xyz><vxyz> 10 0 0</vxyz><charge> 1 </charge>
<xyz> 2.99999  0   -9.08203e-06  </xyz><vxyz>0 0 0  </vxyz><charge>-1 </charge>
<xyz> 0.999    0     3           </xyz><vxyz>0 0 0  </vxyz><charge> 1 </charge>
<xyz> 3        0.001 -1.99999    </xyz><vxyz>0 0 0  </vxyz><charge>-1 </charge>
</coords>
