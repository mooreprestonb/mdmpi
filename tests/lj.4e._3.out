Getting initial energy and force
Starting MD ntime = 1000 with 12 dof
<step>        0 
<energyAVG>
<ham>  4.92946e+01 ~  4.92946e+01 </ham>
<ke>  5.00000e+01 ~  5.00000e+01 </ke> 
<pe> -7.05379e-01 ~ -7.05379e-01 </pe>
</energyAVG>
</step>
<step>       10 
<energyAVG>
<ham>  4.92484e+01 ~  4.88608e+01 </ham>
<ke>  5.04838e+01 ~  4.98530e+01 </ke> 
<pe> -1.23536e+00 ~ -9.92195e-01 </pe>
</energyAVG>
</step>
<step>       20 
<energyAVG>
<ham>  4.89800e+01 ~  4.87663e+01 </ham>
<ke>  4.78928e+01 ~  4.99751e+01 </ke> 
<pe>  1.08716e+00 ~ -1.20875e+00 </pe>
</energyAVG>
</step>
<step>       30 
<energyAVG>
<ham>  4.89102e+01 ~  4.87629e+01 </ham>
<ke>  4.84545e+01 ~  4.95145e+01 </ke> 
<pe>  4.55655e-01 ~ -7.51612e-01 </pe>
</energyAVG>
</step>
<step>       40 
<energyAVG>
<ham>  4.88759e+01 ~  4.87740e+01 </ham>
<ke>  4.87056e+01 ~  4.94777e+01 </ke> 
<pe>  1.70227e-01 ~ -7.03777e-01 </pe>
</energyAVG>
</step>
<step>       50 
<energyAVG>
<ham>  4.88559e+01 ~  4.87739e+01 </ham>
<ke>  4.88592e+01 ~  4.95024e+01 </ke> 
<pe> -3.27523e-03 ~ -7.28414e-01 </pe>
</energyAVG>
</step>
<step>       60 
<energyAVG>
<ham>  4.88425e+01 ~  4.87739e+01 </ham>
<ke>  4.89691e+01 ~  4.95568e+01 </ke> 
<pe> -1.26668e-01 ~ -7.82829e-01 </pe>
</energyAVG>
</step>
<step>       70 
<energyAVG>
<ham>  4.88328e+01 ~  4.87740e+01 </ham>
<ke>  4.90581e+01 ~  4.96425e+01 </ke> 
<pe> -2.25317e-01 ~ -8.68510e-01 </pe>
</energyAVG>
</step>
<step>       80 
<energyAVG>
<ham>  4.88252e+01 ~  4.87686e+01 </ham>
<ke>  4.91387e+01 ~  4.97804e+01 </ke> 
<pe> -3.13519e-01 ~ -1.01182e+00 </pe>
</energyAVG>
</step>
<step>       90 
<energyAVG>
<ham>  4.88188e+01 ~  4.87600e+01 </ham>
<ke>  4.92292e+01 ~  5.02300e+01 </ke> 
<pe> -4.10349e-01 ~ -1.46991e+00 </pe>
</energyAVG>
</step>
<step>      100 
<energyAVG>
<ham>  4.87992e+01 ~  4.79064e+01 </ham>
<ke>  4.89043e+01 ~  4.40175e+01 </ke> 
<pe> -1.05123e-01 ~  3.88895e+00 </pe>
</energyAVG>
</step>
<step>      110 
<energyAVG>
<ham>  4.88029e+01 ~  4.88786e+01 </ham>
<ke>  4.91052e+01 ~  5.10676e+01 </ke> 
<pe> -3.02321e-01 ~ -2.18901e+00 </pe>
</energyAVG>
</step>
<step>      120 
<energyAVG>
<ham>  4.88091e+01 ~  4.88798e+01 </ham>
<ke>  4.92449e+01 ~  5.04130e+01 </ke> 
<pe> -4.35790e-01 ~ -1.53324e+00 </pe>
</energyAVG>
</step>
<step>      130 
<energyAVG>
<ham>  4.88149e+01 ~  4.88849e+01 </ham>
<ke>  4.93070e+01 ~  4.98788e+01 </ke> 
<pe> -4.92142e-01 ~ -9.93909e-01 </pe>
</energyAVG>
</step>
<step>      140 
<energyAVG>
<ham>  4.88201e+01 ~  4.88902e+01 </ham>
<ke>  4.93421e+01 ~  4.97671e+01 </ke> 
<pe> -5.22017e-01 ~ -8.76890e-01 </pe>
</energyAVG>
</step>
<step>      150 
<energyAVG>
<ham>  4.88251e+01 ~  4.88953e+01 </ham>
<ke>  4.93700e+01 ~  4.97658e+01 </ke> 
<pe> -5.44893e-01 ~ -8.70447e-01 </pe>
</energyAVG>
</step>
<step>      160 
<energyAVG>
<ham>  4.88294e+01 ~  4.88953e+01 </ham>
<ke>  4.93952e+01 ~  4.97831e+01 </ke> 
<pe> -5.65739e-01 ~ -8.87860e-01 </pe>
</energyAVG>
</step>
<step>      170 
<energyAVG>
<ham>  4.88333e+01 ~  4.88953e+01 </ham>
<ke>  4.94179e+01 ~  4.97786e+01 </ke> 
<pe> -5.84595e-01 ~ -8.83296e-01 </pe>
</energyAVG>
</step>
<step>      180 
<energyAVG>
<ham>  4.88367e+01 ~  4.88953e+01 </ham>
<ke>  4.94368e+01 ~  4.97400e+01 </ke> 
<pe> -6.00062e-01 ~ -8.44757e-01 </pe>
</energyAVG>
</step>
<step>      190 
<energyAVG>
<ham>  4.88398e+01 ~  4.88953e+01 </ham>
<ke>  4.94510e+01 ~  4.96833e+01 </ke> 
<pe> -6.11261e-01 ~ -7.87942e-01 </pe>
</energyAVG>
</step>
<step>      200 
<energyAVG>
<ham>  4.88424e+01 ~  4.88898e+01 </ham>
<ke>  4.94613e+01 ~  4.96415e+01 </ke> 
<pe> -6.18864e-01 ~ -7.51661e-01 </pe>
</energyAVG>
</step>
<step>      210 
<energyAVG>
<ham>  4.88444e+01 ~  4.88842e+01 </ham>
<ke>  4.94708e+01 ~  4.97126e+01 </ke> 
<pe> -6.26355e-01 ~ -8.28405e-01 </pe>
</energyAVG>
</step>
<step>      220 
<energyAVG>
<ham>  4.88463e+01 ~  4.88730e+01 </ham>
<ke>  4.95006e+01 ~  5.09103e+01 </ke> 
<pe> -6.54287e-01 ~ -2.03736e+00 </pe>
</energyAVG>
</step>
<step>      230 
<energyAVG>
<ham>  4.88346e+01 ~  4.86890e+01 </ham>
<ke>  4.93326e+01 ~  5.02296e+01 </ke> 
<pe> -4.98021e-01 ~ -1.54059e+00 </pe>
</energyAVG>
</step>
<step>      240 
<energyAVG>
<ham>  4.88284e+01 ~  4.86839e+01 </ham>
<ke>  4.93409e+01 ~  4.92860e+01 </ke> 
<pe> -5.12547e-01 ~ -6.02026e-01 </pe>
</energyAVG>
</step>
<step>      250 
<energyAVG>
<ham>  4.88227e+01 ~  4.86893e+01 </ham>
<ke>  4.93349e+01 ~  4.91381e+01 </ke> 
<pe> -5.12155e-01 ~ -4.48836e-01 </pe>
</energyAVG>
</step>
<step>      260 
<energyAVG>
<ham>  4.88176e+01 ~  4.86893e+01 </ham>
<ke>  4.93265e+01 ~  4.91079e+01 </ke> 
<pe> -5.08855e-01 ~ -4.18643e-01 </pe>
</energyAVG>
</step>
<step>      270 
<energyAVG>
<ham>  4.88129e+01 ~  4.86893e+01 </ham>
<ke>  4.93190e+01 ~  4.91471e+01 </ke> 
<pe> -5.06117e-01 ~ -4.57868e-01 </pe>
</energyAVG>
</step>
<step>      280 
<energyAVG>
<ham>  4.88085e+01 ~  4.86893e+01 </ham>
<ke>  4.93149e+01 ~  4.92615e+01 </ke> 
<pe> -5.06386e-01 ~ -5.72193e-01 </pe>
</energyAVG>
</step>
<step>      290 
<energyAVG>
<ham>  4.88043e+01 ~  4.86839e+01 </ham>
<ke>  4.93168e+01 ~  4.94834e+01 </ke> 
<pe> -5.12526e-01 ~ -7.99446e-01 </pe>
</energyAVG>
</step>
<step>      300 
<energyAVG>
<ham>  4.88001e+01 ~  4.86783e+01 </ham>
<ke>  4.93290e+01 ~  4.98680e+01 </ke> 
<pe> -5.28811e-01 ~ -1.18969e+00 </pe>
</energyAVG>
</step>
<step>      310 
<energyAVG>
<ham>  4.87962e+01 ~  4.86750e+01 </ham>
<ke>  4.93496e+01 ~  4.98022e+01 </ke> 
<pe> -5.53439e-01 ~ -1.12728e+00 </pe>
</energyAVG>
</step>
<step>      320 
<energyAVG>
<ham>  4.87925e+01 ~  4.86751e+01 </ham>
<ke>  4.93521e+01 ~  4.97427e+01 </ke> 
<pe> -5.59564e-01 ~ -1.06763e+00 </pe>
</energyAVG>
</step>
<step>      330 
<energyAVG>
<ham>  4.87892e+01 ~  4.86841e+01 </ham>
<ke>  4.93675e+01 ~  4.97411e+01 </ke> 
<pe> -5.78330e-01 ~ -1.05703e+00 </pe>
</energyAVG>
</step>
<step>      340 
<energyAVG>
<ham>  4.87862e+01 ~  4.86897e+01 </ham>
<ke>  4.93727e+01 ~  4.94161e+01 </ke> 
<pe> -5.86519e-01 ~ -7.26348e-01 </pe>
</energyAVG>
</step>
<step>      350 
<energyAVG>
<ham>  4.87834e+01 ~  4.86841e+01 </ham>
<ke>  4.93714e+01 ~  4.92751e+01 </ke> 
<pe> -5.88023e-01 ~ -5.91007e-01 </pe>
</energyAVG>
</step>
<step>      360 
<energyAVG>
<ham>  4.87806e+01 ~  4.86841e+01 </ham>
<ke>  4.93680e+01 ~  4.92418e+01 </ke> 
<pe> -5.87420e-01 ~ -5.57708e-01 </pe>
</energyAVG>
</step>
<step>      370 
<energyAVG>
<ham>  4.87780e+01 ~  4.86785e+01 </ham>
<ke>  4.93650e+01 ~  4.92699e+01 </ke> 
<pe> -5.86988e-01 ~ -5.91376e-01 </pe>
</energyAVG>
</step>
<step>      380 
<energyAVG>
<ham>  4.87754e+01 ~  4.86785e+01 </ham>
<ke>  4.93634e+01 ~  4.93365e+01 </ke> 
<pe> -5.88001e-01 ~ -6.57992e-01 </pe>
</energyAVG>
</step>
<step>      390 
<energyAVG>
<ham>  4.87729e+01 ~  4.86785e+01 </ham>
<ke>  4.93639e+01 ~  4.94272e+01 </ke> 
<pe> -5.91026e-01 ~ -7.48723e-01 </pe>
</energyAVG>
</step>
<step>      400 
<energyAVG>
<ham>  4.87705e+01 ~  4.86785e+01 </ham>
<ke>  4.93669e+01 ~  4.95367e+01 </ke> 
<pe> -5.96418e-01 ~ -8.58224e-01 </pe>
</energyAVG>
</step>
<step>      410 
<energyAVG>
<ham>  4.87683e+01 ~  4.86786e+01 </ham>
<ke>  4.93729e+01 ~  4.96813e+01 </ke> 
<pe> -6.04610e-01 ~ -1.00276e+00 </pe>
</energyAVG>
</step>
<step>      420 
<energyAVG>
<ham>  4.87662e+01 ~  4.86787e+01 </ham>
<ke>  4.93831e+01 ~  4.99293e+01 </ke> 
<pe> -6.16979e-01 ~ -1.25055e+00 </pe>
</energyAVG>
</step>
<step>      430 
<energyAVG>
<ham>  4.87641e+01 ~  4.86788e+01 </ham>
<ke>  4.94018e+01 ~  5.04666e+01 </ke> 
<pe> -6.37714e-01 ~ -1.78786e+00 </pe>
</energyAVG>
</step>
<step>      440 
<energyAVG>
<ham>  4.87621e+01 ~  4.86686e+01 </ham>
<ke>  4.94304e+01 ~  5.02002e+01 </ke> 
<pe> -6.68323e-01 ~ -1.53161e+00 </pe>
</energyAVG>
</step>
<step>      450 
<energyAVG>
<ham>  4.87602e+01 ~  4.86787e+01 </ham>
<ke>  4.94436e+01 ~  5.08779e+01 </ke> 
<pe> -6.83397e-01 ~ -2.19923e+00 </pe>
</energyAVG>
</step>
<step>      460 
<energyAVG>
<ham>  4.87585e+01 ~  4.86842e+01 </ham>
<ke>  4.94684e+01 ~  5.02837e+01 </ke> 
<pe> -7.09848e-01 ~ -1.59950e+00 </pe>
</energyAVG>
</step>
<step>      470 
<energyAVG>
<ham>  4.87570e+01 ~  4.86840e+01 </ham>
<ke>  4.94808e+01 ~  4.99098e+01 </ke> 
<pe> -7.23806e-01 ~ -1.22586e+00 </pe>
</energyAVG>
</step>
<step>      480 
<energyAVG>
<ham>  4.87554e+01 ~  4.86839e+01 </ham>
<ke>  4.94876e+01 ~  4.97445e+01 </ke> 
<pe> -7.32151e-01 ~ -1.06065e+00 </pe>
</energyAVG>
</step>
<step>      490 
<energyAVG>
<ham>  4.87540e+01 ~  4.86839e+01 </ham>
<ke>  4.94918e+01 ~  4.96560e+01 </ke> 
<pe> -7.37776e-01 ~ -9.72184e-01 </pe>
</energyAVG>
</step>
<step>      500 
<energyAVG>
<ham>  4.87526e+01 ~  4.86838e+01 </ham>
<ke>  4.94943e+01 ~  4.95953e+01 </ke> 
<pe> -7.41763e-01 ~ -9.11455e-01 </pe>
</energyAVG>
</step>
<step>      510 
<energyAVG>
<ham>  4.87512e+01 ~  4.86838e+01 </ham>
<ke>  4.94958e+01 ~  4.95449e+01 </ke> 
<pe> -7.44529e-01 ~ -8.61093e-01 </pe>
</energyAVG>
</step>
<step>      520 
<energyAVG>
<ham>  4.87499e+01 ~  4.86839e+01 </ham>
<ke>  4.94963e+01 ~  4.95066e+01 </ke> 
<pe> -7.46334e-01 ~ -8.22781e-01 </pe>
</energyAVG>
</step>
<step>      530 
<energyAVG>
<ham>  4.87487e+01 ~  4.86839e+01 </ham>
<ke>  4.94963e+01 ~  4.94914e+01 </ke> 
<pe> -7.47573e-01 ~ -8.07502e-01 </pe>
</energyAVG>
</step>
<step>      540 
<energyAVG>
<ham>  4.87476e+01 ~  4.86893e+01 </ham>
<ke>  4.94963e+01 ~  4.95034e+01 </ke> 
<pe> -7.48694e-01 ~ -8.14081e-01 </pe>
</energyAVG>
</step>
<step>      550 
<energyAVG>
<ham>  4.87465e+01 ~  4.86893e+01 </ham>
<ke>  4.94967e+01 ~  4.95316e+01 </ke> 
<pe> -7.50149e-01 ~ -8.42314e-01 </pe>
</energyAVG>
</step>
<step>      560 
<energyAVG>
<ham>  4.87455e+01 ~  4.86893e+01 </ham>
<ke>  4.94976e+01 ~  4.95650e+01 </ke> 
<pe> -7.52116e-01 ~ -8.75656e-01 </pe>
</energyAVG>
</step>
<step>      570 
<energyAVG>
<ham>  4.87445e+01 ~  4.86893e+01 </ham>
<ke>  4.94991e+01 ~  4.96000e+01 </ke> 
<pe> -7.54614e-01 ~ -9.10710e-01 </pe>
</energyAVG>
</step>
<step>      580 
<energyAVG>
<ham>  4.87436e+01 ~  4.86893e+01 </ham>
<ke>  4.95012e+01 ~  4.96388e+01 </ke> 
<pe> -7.57660e-01 ~ -9.49476e-01 </pe>
</energyAVG>
</step>
<step>      590 
<energyAVG>
<ham>  4.87426e+01 ~  4.86893e+01 </ham>
<ke>  4.95040e+01 ~  4.96833e+01 </ke> 
<pe> -7.61313e-01 ~ -9.94035e-01 </pe>
</energyAVG>
</step>
<step>      600 
<energyAVG>
<ham>  4.87417e+01 ~  4.86839e+01 </ham>
<ke>  4.95074e+01 ~  4.97320e+01 </ke> 
<pe> -7.65669e-01 ~ -1.04814e+00 </pe>
</energyAVG>
</step>
<step>      610 
<energyAVG>
<ham>  4.87408e+01 ~  4.86839e+01 </ham>
<ke>  4.95115e+01 ~  4.97878e+01 </ke> 
<pe> -7.70767e-01 ~ -1.10386e+00 </pe>
</energyAVG>
</step>
<step>      620 
<energyAVG>
<ham>  4.87398e+01 ~  4.86841e+01 </ham>
<ke>  4.95169e+01 ~  4.99131e+01 </ke> 
<pe> -7.77126e-01 ~ -1.22902e+00 </pe>
</energyAVG>
</step>
<step>      630 
<energyAVG>
<ham>  4.87389e+01 ~  4.86895e+01 </ham>
<ke>  4.95256e+01 ~  5.02288e+01 </ke> 
<pe> -7.86661e-01 ~ -1.53925e+00 </pe>
</energyAVG>
</step>
<step>      640 
<energyAVG>
<ham>  4.87382e+01 ~  4.86878e+01 </ham>
<ke>  4.95412e+01 ~  5.07721e+01 </ke> 
<pe> -8.03073e-01 ~ -2.08426e+00 </pe>
</energyAVG>
</step>
<step>      650 
<energyAVG>
<ham>  4.87374e+01 ~  4.86938e+01 </ham>
<ke>  4.95542e+01 ~  4.99353e+01 </ke> 
<pe> -8.16784e-01 ~ -1.24152e+00 </pe>
</energyAVG>
</step>
<step>      660 
<energyAVG>
<ham>  4.87366e+01 ~  4.86886e+01 </ham>
<ke>  4.95705e+01 ~  5.07847e+01 </ke> 
<pe> -8.33883e-01 ~ -2.09611e+00 </pe>
</energyAVG>
</step>
<step>      670 
<energyAVG>
<ham>  4.87359e+01 ~  4.86895e+01 </ham>
<ke>  4.95845e+01 ~  5.03111e+01 </ke> 
<pe> -8.48590e-01 ~ -1.62164e+00 </pe>
</energyAVG>
</step>
<step>      680 
<energyAVG>
<ham>  4.87352e+01 ~  4.86838e+01 </ham>
<ke>  4.95931e+01 ~  5.00910e+01 </ke> 
<pe> -8.57885e-01 ~ -1.40724e+00 </pe>
</energyAVG>
</step>
<step>      690 
<energyAVG>
<ham>  4.87345e+01 ~  4.86852e+01 </ham>
<ke>  4.96007e+01 ~  5.02417e+01 </ke> 
<pe> -8.66246e-01 ~ -1.55651e+00 </pe>
</energyAVG>
</step>
<step>      700 
<energyAVG>
<ham>  4.87335e+01 ~  4.86629e+01 </ham>
<ke>  4.96138e+01 ~  4.87208e+01 </ke> 
<pe> -8.80300e-01 ~ -5.79660e-02 </pe>
</energyAVG>
</step>
<step>      710 
<energyAVG>
<ham>  4.87327e+01 ~  4.86815e+01 </ham>
<ke>  4.96194e+01 ~  5.04555e+01 </ke> 
<pe> -8.86662e-01 ~ -1.77396e+00 </pe>
</energyAVG>
</step>
<step>      720 
<energyAVG>
<ham>  4.87320e+01 ~  4.86788e+01 </ham>
<ke>  4.96263e+01 ~  5.00535e+01 </ke> 
<pe> -8.94331e-01 ~ -1.37467e+00 </pe>
</energyAVG>
</step>
<step>      730 
<energyAVG>
<ham>  4.87314e+01 ~  4.86899e+01 </ham>
<ke>  4.96342e+01 ~  5.03953e+01 </ke> 
<pe> -9.02839e-01 ~ -1.70538e+00 </pe>
</energyAVG>
</step>
<step>      740 
<energyAVG>
<ham>  4.87307e+01 ~  4.86704e+01 </ham>
<ke>  4.96483e+01 ~  5.06565e+01 </ke> 
<pe> -9.17518e-01 ~ -1.98614e+00 </pe>
</energyAVG>
</step>
<step>      750 
<energyAVG>
<ham>  4.87300e+01 ~  4.86696e+01 </ham>
<ke>  4.96516e+01 ~  5.05814e+01 </ke> 
<pe> -9.21578e-01 ~ -1.91181e+00 </pe>
</energyAVG>
</step>
<step>      760 
<energyAVG>
<ham>  4.87293e+01 ~  4.86789e+01 </ham>
<ke>  4.96651e+01 ~  5.04361e+01 </ke> 
<pe> -9.35743e-01 ~ -1.75718e+00 </pe>
</energyAVG>
</step>
<step>      770 
<energyAVG>
<ham>  4.87287e+01 ~  4.86788e+01 </ham>
<ke>  4.96717e+01 ~  5.00231e+01 </ke> 
<pe> -9.43084e-01 ~ -1.34426e+00 </pe>
</energyAVG>
</step>
<step>      780 
<energyAVG>
<ham>  4.87280e+01 ~  4.86787e+01 </ham>
<ke>  4.96748e+01 ~  4.98426e+01 </ke> 
<pe> -9.46805e-01 ~ -1.16386e+00 </pe>
</energyAVG>
</step>
<step>      790 
<energyAVG>
<ham>  4.87274e+01 ~  4.86786e+01 </ham>
<ke>  4.96762e+01 ~  4.97474e+01 </ke> 
<pe> -9.48842e-01 ~ -1.06874e+00 </pe>
</energyAVG>
</step>
<step>      800 
<energyAVG>
<ham>  4.87268e+01 ~  4.86786e+01 </ham>
<ke>  4.96766e+01 ~  4.96748e+01 </ke> 
<pe> -9.49833e-01 ~ -9.96195e-01 </pe>
</energyAVG>
</step>
<step>      810 
<energyAVG>
<ham>  4.87262e+01 ~  4.86786e+01 </ham>
<ke>  4.96761e+01 ~  4.96074e+01 </ke> 
<pe> -9.49942e-01 ~ -9.28785e-01 </pe>
</energyAVG>
</step>
<step>      820 
<energyAVG>
<ham>  4.87256e+01 ~  4.86786e+01 </ham>
<ke>  4.96749e+01 ~  4.95506e+01 </ke> 
<pe> -9.49288e-01 ~ -8.72002e-01 </pe>
</energyAVG>
</step>
<step>      830 
<energyAVG>
<ham>  4.87250e+01 ~  4.86786e+01 </ham>
<ke>  4.96731e+01 ~  4.95115e+01 </ke> 
<pe> -9.48080e-01 ~ -8.32819e-01 </pe>
</energyAVG>
</step>
<step>      840 
<energyAVG>
<ham>  4.87245e+01 ~  4.86786e+01 </ham>
<ke>  4.96711e+01 ~  4.94905e+01 </ke> 
<pe> -9.46556e-01 ~ -8.11842e-01 </pe>
</energyAVG>
</step>
<step>      850 
<energyAVG>
<ham>  4.87240e+01 ~  4.86786e+01 </ham>
<ke>  4.96689e+01 ~  4.94834e+01 </ke> 
<pe> -9.44916e-01 ~ -8.04711e-01 </pe>
</energyAVG>
</step>
<step>      860 
<energyAVG>
<ham>  4.87234e+01 ~  4.86786e+01 </ham>
<ke>  4.96667e+01 ~  4.94842e+01 </ke> 
<pe> -9.43288e-01 ~ -8.05545e-01 </pe>
</energyAVG>
</step>
<step>      870 
<energyAVG>
<ham>  4.87229e+01 ~  4.86786e+01 </ham>
<ke>  4.96646e+01 ~  4.94870e+01 </ke> 
<pe> -9.41726e-01 ~ -8.08376e-01 </pe>
</energyAVG>
</step>
<step>      880 
<energyAVG>
<ham>  4.87224e+01 ~  4.86786e+01 </ham>
<ke>  4.96626e+01 ~  4.94849e+01 </ke> 
<pe> -9.40208e-01 ~ -8.06303e-01 </pe>
</energyAVG>
</step>
<step>      890 
<energyAVG>
<ham>  4.87219e+01 ~  4.86786e+01 </ham>
<ke>  4.96605e+01 ~  4.94686e+01 </ke> 
<pe> -9.38623e-01 ~ -7.89986e-01 </pe>
</energyAVG>
</step>
<step>      900 
<energyAVG>
<ham>  4.87214e+01 ~  4.86733e+01 </ham>
<ke>  4.96582e+01 ~  4.94354e+01 </ke> 
<pe> -9.36809e-01 ~ -7.62103e-01 </pe>
</energyAVG>
</step>
<step>      910 
<energyAVG>
<ham>  4.87208e+01 ~  4.86680e+01 </ham>
<ke>  4.96556e+01 ~  4.94196e+01 </ke> 
<pe> -9.34780e-01 ~ -7.51632e-01 </pe>
</energyAVG>
</step>
<step>      920 
<energyAVG>
<ham>  4.87203e+01 ~  4.86735e+01 </ham>
<ke>  4.96532e+01 ~  4.94496e+01 </ke> 
<pe> -9.32900e-01 ~ -7.76107e-01 </pe>
</energyAVG>
</step>
<step>      930 
<energyAVG>
<ham>  4.87198e+01 ~  4.86734e+01 </ham>
<ke>  4.96513e+01 ~  4.94904e+01 </ke> 
<pe> -9.31461e-01 ~ -8.17013e-01 </pe>
</energyAVG>
</step>
<step>      940 
<energyAVG>
<ham>  4.87193e+01 ~  4.86789e+01 </ham>
<ke>  4.96497e+01 ~  4.95008e+01 </ke> 
<pe> -9.30325e-01 ~ -8.21932e-01 </pe>
</energyAVG>
</step>
<step>      950 
<energyAVG>
<ham>  4.87189e+01 ~  4.86789e+01 </ham>
<ke>  4.96480e+01 ~  4.94780e+01 </ke> 
<pe> -9.29071e-01 ~ -7.99133e-01 </pe>
</energyAVG>
</step>
<step>      960 
<energyAVG>
<ham>  4.87185e+01 ~  4.86843e+01 </ham>
<ke>  4.96461e+01 ~  4.94538e+01 </ke> 
<pe> -9.27526e-01 ~ -7.69463e-01 </pe>
</energyAVG>
</step>
<step>      970 
<energyAVG>
<ham>  4.87182e+01 ~  4.86843e+01 </ham>
<ke>  4.96440e+01 ~  4.94404e+01 </ke> 
<pe> -9.25810e-01 ~ -7.56053e-01 </pe>
</energyAVG>
</step>
<step>      980 
<energyAVG>
<ham>  4.87179e+01 ~  4.86840e+01 </ham>
<ke>  4.96419e+01 ~  4.94462e+01 </ke> 
<pe> -9.24066e-01 ~ -7.62266e-01 </pe>
</energyAVG>
</step>
<step>      990 
<energyAVG>
<ham>  4.87175e+01 ~  4.86786e+01 </ham>
<ke>  4.96402e+01 ~  4.94991e+01 </ke> 
<pe> -9.22710e-01 ~ -8.20426e-01 </pe>
</energyAVG>
</step>
<step>     1000 
<energyAVG>
<ham>  4.87171e+01 ~  4.86787e+01 </ham>
<ke>  4.96393e+01 ~  4.96066e+01 </ke> 
<pe> -9.22239e-01 ~ -9.27867e-01 </pe>
</energyAVG>
</step>

